projects/find/find.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+import argparse
+
+
+def find(path, name=None, out=sys.stdout):
+    """This method is created to find all files and directories, given a specific path.
+    Optionally, your can use -name flag to search only the files that match to a specific name"""
+    if not os.path.exists(path):
+        raise ValueError("Path doesn't exist or wrong: please try again")
+    stack = [path]
+    while len(stack) > 0:
+        element = stack.pop()
+        # decided to follow the same implementation of real find in which symlink are not navigate through
```

Comment:
Is this a decision you should take? Did the client take part in the decision process??